### PR TITLE
🔧 cookiebanner & footer opacity

### LINKED ISSUE #3903: Cookie banner opacity causes content overlap on mobile viewport
### What happened?

`We use third-party 🥠  in order to personalize your site experience    ACCEPT     DECLINE`

The cookie banner's opacity make its content overlap other contents on mobile viewport. 

Issue is present in all mobile browsers — Chrome, Safari, DuckDuckGo, Brave, Kiwi and UC browser — on which I opened kodadot.xyz and beta.kodadot.xyz

Issue is also present when kodadot.xyz is opened in desktop browsers, but as desktop browsers have a wider viewport, it's almost unnoticeable and less clumsy. 

Issue is present in both `white` and `dark` mode.

### Please reproduce in steps

> Visit kodadot.xyz or beta.kodadot.xyz as a first time user or in a "fresh" browser

> See the footer part of page, the cookie banner content's opacity settings make it overlap other contents.

### Expected Behavior

Cookie banner and its content are `NOT` to overlap other contents

### What browsers are you seeing the problem on?

Mobile Android Chrome, Mobile iOS Safari, Mobile Firefox

### At which address did you encounter bug?

beta.kodadot.xyz, kodadot.xyz

### Are you logged in?

_No response_

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

Screenshot_20220902-115135_DuckDuckGo
Screenshot_20220902-114854_Brave

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by roiLeo**
### PR type

- [x] Bugfix

### What's new?

- [x] PR closes #3903 
- [x] removed opacity on cookie banner

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/CookieBanner.vue [hunk 1]
```diff
@@ -27,7 +27,7 @@ import { Component, Vue } from 'nuxt-property-decorator'
   components: {}, // do not remove!
 })
 export default class CookieBanner extends Vue {
-  protected hasDisplayedCookieBanner: boolean =
+  public hasDisplayedCookieBanner: boolean =
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/CookieBanner.vue:30]: > just a technical question: is localStorage even available when the component properties are being initialized? 

Yes? or else it wouldn't work as planned. I think it's accessible just before `mounted()`

> is localstorage available at this time?

Yes but not on SSR

**Review by petersopko — APPROVED** [review]
wfm

**Comment by petersopko** [comment]
Cookie banner is something which always bothered me on our mobile and I think it deserves some rethinking, but let's wait until new landing comes and perhaps @exezbcz will have some further thoughts on the subject

File: components/CookieBanner.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (petersopko) [reply components/CookieBanner.vue:30]: @kkukelka it works as expected, I hope you don't mind me resolving this conversation in favor of merging :)

**Comment by roiLeo** [comment]
> Cookie banner is something which always bothered me on our mobile and I think it deserves some rethinking, but let's wait until new landing comes and perhaps @exezbcz will have some further thoughts on the subject

This is mandatory, we're still mising a cookie policy and I'm pretty sure we're not in compliance with EU rules

File: components/CookieBanner.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (kkukelka) [reply components/CookieBanner.vue:30]: sure no worries

**Pull request merged**